deploy.gradle

```diff
@@ -55,6 +55,7 @@ subprojects {
       task.from(rootDir) {
         include 'LICENSE'
         include 'NOTICE'
+        include 'iceberg-version.properties'
```

Comment:
I think I said this should be `iceberg-build.properties`.

Reply:
I will update that.

Reply:
This file won't exist in the tarball as the PR is (with the properties file in the requested `build/iceberg-version.properties`), so I've removed this line from deploy.gradle and we can add it in a subsequent PR.